src/libraries/Microsoft.PowerFx.Core/Parser/ParseResult.cs

```diff
@@ -44,11 +45,19 @@ public class ParseResult : IOperationStatus
         internal SourceList Before { get; }
 
         internal SourceList After { get; }
-        
+
+        internal CultureInfo Locale { get; }
```

Comment:
>internal CultureInfo Locale { get; } [](http://example.com/codeflow?start=8&length=36)

We need to be clear that this is the locale we show error messages in; not the locale that we lexed in. 

Maybe we need to rename, like "ErrorMessageLocale"

We have 3 cultures:

1. How we lex in 
2. What compile-time error messages we show 
3. default for Runtime  (when no other culture is specified)